README.md

```diff
@@ -30,6 +30,7 @@ If you are reading this it looks like you are looking to add an egg to your serv
 * [Bastion](/bots/discord/bastion) Node JS
 * [CorpBot](/bots/discord/corpbot) Python
 * [discord.js](bots/discord/discord.js) Node JS generic
+* [nodemon.js](bots/discord/nodemon.js) Node JS generic
```

Comment:
Readme is supposed to be in alphabetical order nor should this have a duplicate name. Call this nodemon JS